model/atmosphere/dycore/tests/dycore_stencil_tests/test_fused_velocity_advection_stencil_15_to_18.py

```diff
@@ -185,33 +180,33 @@ def reference(
 
     @pytest.fixture
     def input_data(self, grid):
-        z_w_con_c = random_field(grid, dims.CellDim, dims.KDim, extend={dims.KDim: 1})
-        w = random_field(grid, dims.CellDim, dims.KDim, extend={dims.KDim: 1})
-        coeff1_dwdz = random_field(grid, dims.CellDim, dims.KDim)
-        coeff2_dwdz = random_field(grid, dims.CellDim, dims.KDim)
+        z_w_con_c = data_alloc.random_field(grid, dims.CellDim, dims.KDim, extend={dims.KDim: 1})
+        w = data_alloc.random_field(grid, dims.CellDim, dims.KDim, extend={dims.KDim: 1})
+        coeff1_dwdz = data_alloc.random_field(grid, dims.CellDim, dims.KDim)
+        coeff2_dwdz = data_alloc.random_field(grid, dims.CellDim, dims.KDim)
 
-        z_v_grad_w = random_field(grid, dims.EdgeDim, dims.KDim)
-        e_bln_c_s = as_1D_sparse_field(random_field(grid, dims.CellDim, dims.C2EDim), dims.CEDim)
+        z_v_grad_w = data_alloc.random_field(grid, dims.EdgeDim, dims.KDim)
+        e_bln_c_s = data_alloc.random_field(grid, dims.CEDim)
 
-        levelmask = random_mask(grid, dims.KDim)
-        cfl_clipping = random_mask(grid, dims.CellDim, dims.KDim)
-        owner_mask = random_mask(grid, dims.CellDim)
-        ddqz_z_half = random_field(grid, dims.CellDim, dims.KDim)
-        area = random_field(grid, dims.CellDim)
-        geofac_n2s = random_field(grid, dims.CellDim, dims.C2E2CODim)
+        levelmask = data_alloc.random_mask(grid, dims.KDim)
+        cfl_clipping = data_alloc.random_mask(grid, dims.CellDim, dims.KDim)
+        owner_mask = data_alloc.random_mask(grid, dims.CellDim)
+        ddqz_z_half = data_alloc.random_field(grid, dims.CellDim, dims.KDim)
+        area = data_alloc.random_field(grid, dims.CellDim)
+        geofac_n2s = data_alloc.random_field(grid, dims.CellDim, dims.C2E2CODim)
 
-        z_w_con_c_full = zero_field(grid, dims.CellDim, dims.KDim)
-        ddt_w_adv = zero_field(grid, dims.CellDim, dims.KDim)
+        z_w_con_c_full = data_alloc.zero_field(grid, dims.CellDim, dims.KDim)
+        ddt_w_adv = data_alloc.zero_field(grid, dims.CellDim, dims.KDim)
 
         scalfac_exdiff = 10.0
         cfl_w_limit = 3.0
         dtime = 2.0
 
-        k = zero_field(grid, dims.KDim, dtype=gtx.int32)
+        k = data_alloc.zero_field(grid, dims.KDim, dtype=gtx.int32)
```

Comment:
ok I had a though: it would be more consistent if we stick to use types imported from type_alias.py, which in this case would mean adding int32 import in that file: 
```python
from gt4py.next.ffront.fbuiltins import float32, float64, int32
```

What do you think?

Reply:
I don't know. We use the import of the `gt4py` simple types every where and in case we decide to ditch the mixed precision we could simply drop the `type_alias.py`..